packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -35,10 +44,31 @@ export class DemoComponent implements OnDestroy, OnInit {
   selectedOutputFormat: SupportedOutputs | string = "html";
   language: "eng" | "fra" | "spa" = "eng";
   unsubscribe$ = new Subject<void>();
+  xmlSerializer = new XMLSerializer();
+  readmeFile = new Blob(
+    [
+      `Web Deployment Guide
+
+This bundle has everything you need to host your ReadAlong on your own server.
+
+Your audio, (optional) image, and alignment (.readalong) assets are stored in the assets folder.
+
+The plain text used to create your ReadAlong is also stored here along with an example index.html file.
+
+Your index.html file demonstrates the snippet and imports needed to host the ReadAlong on your server.
+
+Please host all assets on your server, include the font and package imports defined in the index.html in your website's imports, and include the corresponding <readalong> snippet everywhere you would like your ReadAlong to be displayed.
```

Comment:
That should say `<read-along>`, not `<readalong>`. It look me a while to notice this, but it's been nagging me, something felt wrong but I didn't quite know what... :)